### PR TITLE
use kebab-case consistently throughout project

### PR DESCRIPTION
**Pull request by SunburtReynolds**
### Description

As per @rgerstenberger's comment on another PR, we want to use `kebab-case` consistently to avoid cross-platform file system issues.

#### Type of Change

- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?

I retested CLI functionality, and everything is normal.

### CODE REVIEW

**Comment by narinluangrath** [comment]
LGTM

File: examples/MDX/test-mdx.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import slides from './slides.mdx';
-import Deck from '../../src/components/Deck';
-import Slide from '../../src/components/Slide';
+import Deck from '../../src/components/deck';
```
Comment (rgerstenberger) [inline examples/MDX/test-mdx.js]: This is still `/MDX/*` which doesn't solve the problem.

Comment (SunburtReynolds) [reply examples/MDX/test-mdx.js]: It's lower case locally. I tried to make it upper then lower case again. No diff in git.

Screen Shot 2019-08-12 at 3 22 34 PM

Comment (SunburtReynolds) [reply examples/MDX/test-mdx.js]: Fixed!

**Pull request merged**